src/components/shapes/draw.js

```diff
@@ -141,6 +140,8 @@ function setupDragElement(gd, shapePath, shapeOptions, index) {
 
     dragElement.init(dragOptions);
 
+    shapePath.node().onmousemove = updateDragMode;
```

Comment:
Referencing the ticket for annotation edits -> https://github.com/plotly/plotly.js/issues/679